qa/smoke-test-reindex-with-all-modules/src/test/resources/rest-api-spec/test/reindex/30_timeout.yml

```diff
@@ -1,33 +0,0 @@
----
-"Timeout":
-  - skip:
-      version: "all"
-      reason:  painless doesn't support thread.sleep so we need to come up with a better way
```

Comment:
We haven't been running this for *forever* and we've been ok so I just dropped it.